lib/reporter/json_reporter.js

```diff
@@ -0,0 +1,77 @@
+// {
+//   "file": "file",
+//   "issues_count": 3,
+//   "issues" : [
+//     {
+//       "line": 4,
+//       "content": "FROM ...",
+//       "category": "clarity",
+//       "title": "...",
+//       "description": "...."
+//     }, ..., {}
+//   ]
+// }
+
+'use strict';
+
+const notDeepStrictEqual = require('assert').notDeepStrictEqual;
+const Reporter = require('./reporter');
+
+class JsonReporter extends Reporter {
+  constructor (opts) {
+    super();
+    opts = opts || {};
```

Comment:
Since `opts` aren't used in this constructor, this line is kinda pointless. Let's remove it.